User-Stories/Compliance-Manager-User-Stories/ComplianceManager.md

```diff
@@ -0,0 +1,46 @@
+#Compliance Manager
+This describes the role of a Compliance Manager. This is a role with suitable legal understanding so that he will be able to interpret the legal situation and assess the circumstances accordingly so that he will be able to judge the consequences from obligations respectively support teams / POs in identifying suitable measures to remain compliant. According to OCv2.0 specification, this role may be staffed by external resources.
+
+##Responsibilities, tasks and role
+The ```primary goal```of the Compliance Manager should be to achieve compliance. He is the advocate of the 3rd party license holders within the company. 
+
+Thus the role will be held ```accountable``` for 
+* Compliance of all open source used  within the products, that received his approval
+* Suitability of the general|all|project specific black/whitelists 
+* Creation, maintenance and governance of the Open Source Policy 
+
+Typically this will result in the following kind of ```tasks```:
+* Generate, publish, promote and maintain the corporate Open Source Policy
+* Operate (organize, manage agenda, etc.) the Open Source Board meetings
+* Provide support to the projects in determining legal obligations
+* Support teams in open sourcing their own components
+* Review compliance applications and provide approvals, respectively suitable guidance
+
+A Compliance Manager will ```escalate```to the Open Source Board or the Senior Management Sponsor.
```

Comment:
Since we also target organizations which might not be "companies" I would like to have another term then "Senior Managmenet Sponsor"

Reply:
Ok, what about "Senior Sponsor" instead?

Reply:
agree

Reply:
changed accordingly.